docstr_coverage/cli.py

```diff
@@ -148,6 +148,15 @@ def parse_ignore_names_file(ignore_names_file: str) -> tuple:
     help="Ignore docstrings of class definitions",
     show_default=True,
 )
+@click.option(
+    "-r",
```

Comment:
Any reason for the "-r" flag? "-P" seems to make more sense, but I'm open to it if you have a reason for "-r"

Reply:
nope. change it to whatever letter you like

Reply:
Changed in 49abac2